docs/core/tools/dotnet-build.md

```diff
@@ -55,9 +57,9 @@ In order to produce a library, omit the `<OutputType>` property. The main differ
 
 ## Arguments
 
-`PROJECT`
+`PROJECT | SOLUTION`
 
-The project file to build. If a project file is not specified, MSBuild searches the current working directory for a file that has a file extension that ends in *proj* and uses that file.
+The project or solution file to build. If a project or solution file is not specified, MSBuild searches the current working directory for a file that has a file extension that ends in *proj* and uses that file.
```

Comment:
or it uses a *sln* file

Reply:
@rpetrusha should it be specified what the build looks for at first? In other words, if both *proj* and *sln* files present in the folder, which one is picked up by `dotnet build`?

Reply:
@pkulikov Neither is picked up. If there are more than one it errs and tells you to specify which you want to use.